centers/models/center.py

```diff
@@ -8,6 +14,17 @@ class Center(models.Model):
     address = models.CharField(max_length=255, blank=True, null=True)
     phone = models.CharField(max_length=45, blank=True, null=True)
     url = models.URLField(blank=True, null=True)
+    hash_id = models.CharField(
+        max_length=12, unique=True)
 
     def __unicode__(self):
         return unicode(self.name)
+
+
+def update_center_hash_id(sender, instance, created, **kwargs):
```

Comment:
http://igorsobreira.com/2010/09/16/difference-between-one-underline-and-two-underlines-in-python.html